CSS/style.css

```diff
@@ -0,0 +1,903 @@
+  }
+
+  #overlay.active {
+    opacity: 1;
+    pointer-events: all;
+  }
+
+  .buttondesk {
+    display: none;
+  }
+
+  #popupbutton {
+    display: flex;
+    justify-content: start;
+    gap: 7%;
+  }
+
+  .mob-modal-image {
+    width: 100%;
+  }
+
+  .home.active,
+  .form-container.active,
+  .about-myself.active,
+  header.active {
+    filter: blur(6px);
+  }
+
+  hr {
+    flex: 0 0 100%;
+    width: 89%;
+    margin-left: 1%;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .work-img1 img,
+  .work-img2 img,
+  .work-img3 img,
+  .work-img4 img {
+    width: 85%;
+    height: 70%;
+  }
+}
+
+@media screen and (max-width: 980px) {
+  .work-img1 img,
+  .work-img2 img,
+  .work-img3 img,
+  .work-img4 img {
+    width: 87%;
+    height: 75%;
+  }
+}
+
+@media screen and (min-width: 980px) {
+  #home {
+    padding: 160px 260px 90px 130px;
+    height: 70vh;
+    text-align: left;
+    border-bottom-left-radius: 20%;
+    background-color: #fff;
+  }
+}
+
+@media screen and (max-width: 1200px) {
+  .work-img1 img,
+  .work-img2 img,
+  .work-img3 img,
+  .work-img4 img {
+    width: 95%;
+    height: 95%;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #home {
+    background-image: url(../img/home_bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .nav-ul {
+    margin-right: 39px;
+  }
+
+  .home p {
+    width: 100%;
+  }
+
+  .home.active,
+  .form-container.active,
+  .about-myself.active,
+  header.active {
+    filter: blur(5px);
+  }
+
+  .contact-top-p {
+    width: 73%;
+  }
+
+  .hamburger {
+    display: none;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: white;
+  }
+
+  .buttonmob {
+    display: none;
+  }
+
+  .work-img {
+    width: 50%;
+    height: 448px;
+  }
+
+  .containertext2,
+  .containertext4 {
+    width: 50%;
+    margin-right: 20px;
+    margin-left: 6px;
+  }
+
+  .nav-link1,
+  .nav-link2 {
+    display: none;
+  }
+
+  .containertext1,
+  .containertext3 {
+    width: 50%;
+    margin-left: 60px;
+    margin-right: 91px;
+  }
+
+  .grid-item {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .grid-item2,
+  .grid-item4 {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .about-myself {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .about-description {
+    width: 50%;
+    margin-right: 37px;
+    margin-left: 80px;
+  }
+
+  .languages-list li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    margin-right: 12px;
+  }
+
+  .languages-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .form-des p {
+    width: 400px;
+  }
+
+  .shape {
+    display: none;
+  }
+
+  .nav-ul a {
+    color: #344563;
+  }
+
+  .nav-ul a:hover {
+    transform: scale(1.5);
+    color: #2230d2;
+    cursor: pointer;
+  }
+}
+
+/* #form-error {
```

Comment:
- [ ] kindly as suggested by the previous reviewer, it would be better to remove all commented lines in your codes to keep them clean and professional 🙏